spring-boot/autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaMeterBindersTest.java

```diff
@@ -66,14 +80,26 @@ protected void doGet(ServiceRequestContext ctx, HttpRequest req,
     }
 
     @Inject
-    private MetricRegistry metricRegistry;
+    private MeterRegistry registry;
 
     @Test
     public void testJvmMetrics() throws Exception {
-        assertThat(metricRegistry.getMetrics().get("jvm.buffers.direct.count")).isNotNull();
-        assertThat(metricRegistry.getMetrics().get("jvm.classloader.loaded")).isNotNull();
-        assertThat(metricRegistry.getMetrics().get("jvm.memory.heap.max")).isNotNull();
-        assertThat(metricRegistry.getMetrics().get("jvm.threads.daemon.count")).isNotNull();
-        assertThat(metricRegistry.getMetrics().get("system.cpu.load-average")).isNotNull();
+        registry.getMeters().forEach(m -> {
+            final String id = m.getName() +
+                              Streams.stream(m.getTags()).collect(toImmutableMap(Tag::getKey, Tag::getValue));
+            final Map<String, Double> values = measureAll(registry, m.getName(), m.getTags());
+            if (values.size() > 1) {
```

Comment:
Yes we can, but it's not pretty when it's just a single-value meter. `foo=[ bar ]` vs. `foo=bar`